src/test/lib/DataTest.cpp

```diff
@@ -3178,3 +3177,209 @@ QuicTestConnectAndIdleForDestCidChange(
         }
     }
 }
+
+#define BUFFER_SIZE 10000
+#define RELIABLE_SIZE 5000
+#define BUFFER_SIZE_MULTI_SENDS 10000
+#define RELIABLE_SIZE_MULTI_SENDS 20000
+
+
+//
+// These Context Structs are useful helpers for the StreamReliableReset test suite.
+// It keeps track of the order of absolute offsets of all the send requests received, and the total number of bytes received.
+// If everything works, SendCompleteOrder MUST be monotonically increasing.
+//
+struct SendContext {
+    BOOLEAN Successful;
+    uint64_t SeqNum;
+};
+struct StreamReliableReset {
+
+    CxPlatEvent ClientStreamShutdownComplete;
+    CxPlatEvent ServerStreamShutdownComplete;
+    uint64_t ReceivedBufferSize;
+    uint64_t SequenceNum;
+    QUIC_UINT62 ShutdownErrorCode;
+
+
+    static QUIC_STATUS ClientStreamCallback(_In_ MsQuicStream*, _In_opt_ void* ClientContext, _Inout_ QUIC_STREAM_EVENT* Event) {
+        auto TestContext = (StreamReliableReset*)ClientContext;
+        if (Event->Type == QUIC_STREAM_EVENT_SHUTDOWN_COMPLETE) {
+            TestContext->ClientStreamShutdownComplete.Set();
+        }
+            // Get the send context of the Event
+        if (Event->Type == QUIC_STREAM_EVENT_SEND_COMPLETE) {
```

Comment:
```suggestion
        // Get the send context of the Event
        if (Event->Type == QUIC_STREAM_EVENT_SEND_COMPLETE) {
```
But is this comment really necessary?

Reply:
To me 1 month ago it was, now it's not necessary.